config/repos.php

```diff
@@ -7,6 +7,9 @@
         'cachethq',
     ],
     'repos' => [
+        'gurpeetkaits' => [
```

Comment:
Hey! Thanks for the contribution. I'm wondering whether there's a small typo here? Is this supposed to be `gurpreetkaits` instead of `gurpeetkaits`? 😄